tests/nix/test-building/default.nix

```diff
@@ -0,0 +1,27 @@
+let
+  # Pinning nixpkgs to specific release
+  # To get sha256 use "nix-prefetch-git <url> --rev <commit>"
+  commitRev="5574b6a152b1b3ae5f93ba37c4ffd1981f62bf5a";
+  nixpkgs = builtins.fetchTarball {
+    url = "https://github.com/NixOS/nixpkgs/archive/${commitRev}.tar.gz";
+    sha256 = "1pqdddp4aiz726c7qs1dwyfzixi14shp0mbzi1jhapl9hrajfsjg";
+  };
+  pkgs = import nixpkgs { config = { allowUnfree = true; }; };
+
+  # Test that we can actually build
+  test-build = pkgs.runCommand "test-build" { } ''
+    touch $out
+  '';
+
+in
+pkgs.mkShell {
+  buildInputs = with pkgs; [
+    python36Packages.numpy
```

Comment:
I think this means we are building numpy from scratch. This can take a bit of time. We are trying to speed up/keep at current level the runtime of our tests (as it is already pretty long).

I don't think there is a way to test for this without having to build a container image. Maybe we can think of one together? Basically more of a unit test type test than an "end to end" test like this.

If we can't replace this by a unit test it would be good to replace `numpy` with something like [`there`](https://pypi.org/project/there/) which is super quick to install. Or maybe we don't even have to install any additional Python packages?

Reply:
> I think this means we are building numpy from scratch. This can take a bit of time. We are trying to speed up/keep at current level the runtime of our tests (as it is already pretty long).

No, it does not. It is cached by the NixOS organization.